src/components/RocketCard.js

```diff
@@ -0,0 +1,48 @@
+/* eslint-disable react/prop-types */
+import React from 'react';
+import { useDispatch } from 'react-redux';
+import { booking, cancelBooking } from '../redux/rocketSlice';
+
+function RocketCard(props) {
+  const dispatch = useDispatch();
+  const { rocket } = props;
+  const {
+    id, rocketName, description, flickrImages, reserved,
+  } = rocket;
+
+  return (
+    <li className="list-container">
+      <img className="rocket-images" src={flickrImages[0]} alt="rockets" />
+      <div className="heading-rockets">
+        <h3 className="rocket-name">{rocketName}</h3>
+        <p className="paragraph">
+          {reserved ? (
+            <button type="button" className="reserved">
+              Reserved
+            </button>
+          ) : null}
+          {description}
+        </p>
+        {reserved ? (
+          <button
+            type="button"
+            className="cancel"
+            onClick={() => dispatch(cancelBooking(id))}
+          >
+            Cancel Reservation
+          </button>
+        ) : (
+          <button
```

Comment:
- [ ] Please kindly design this section as illustrated in the given design:
1. The active nav-link supposed to be underlined.
2. Your styling for the "Reserved" badge and "Cancel reservation" button is a bit different from the template. 👍

## Given design 
<img width="620" alt="Untitled" src="https://github.com/Uzair-Manzoor/react-capstone/assets/103386183/d4d34adc-d0cf-4eb7-b729-8691459167c1">

## Your design
<img width="906" alt="Untitled" src="https://github.com/Uzair-Manzoor/react-capstone/assets/103386183/34c0ad62-383b-47ac-8f72-d1de955751b3">